modules/repository-s3/licenses/httpclient-NOTICE.txt

```diff
@@ -1,5 +1,6 @@
 Apache HttpComponents Client
-Copyright 1999-2015 The Apache Software Foundation
+Copyright 1999-2016 The Apache Software Foundation
```

Comment:
Did we update a dependency here?

Reply:
No, but for some reason the copyright years were different between `repository-s3` and `repository-url`, so I went with the later year. The checksums of the actual jar files match.